microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -103,6 +103,15 @@ You can delete templates stored on the device and clear the cache data. Go to `e
 You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
 Collections aren't available when using **InPrivate** or **Guest** modes.
 
+<!-- ====================================================================== -->
+## Compose
+
+In the Bing side pane, Compose uses AI to write for you starting from a short prompt and "tone", "format", and "length" selections. Text generated by Compose can be copied or inserted into the text page directly. To access Compose, click on the blue Bing icon in the right hand pane > Compose (requires Bing waitlist approval). Text and selections inputed by the user is sent to an internal Microsoft Large Language Model (LLM).
```

Comment:
Introduce "Bing waitlist approval".  Isn't this ended?
https://www.bing.com/search?q=Bing+waitlist

I don't know what "Bing waitlist approval" means, might Find that in this article. ... "wait" doesn't appear elsewhere in the article.

Reply:
@majenny - you might need to delete " (requires Bing waitlist approval)".  The whole "Bing waitlist" mechanism may have ended, per web search above.

Reply:
@majenny - pls close this comment, it's been fixed at latest https://github.com/MicrosoftDocs/edge-developer/pull/2574/files